help/view-zulip-version.md

```diff
@@ -10,8 +10,8 @@ web app your organization is using.
 Zulip Cloud organizations are always updated to the latest version of Zulip.
 
 [upgrade-zulip]:
-    https://zulip.readthedocs.io/en/latest/production/upgrade-or-modify.html
-[changelog]: https://zulip.readthedocs.io/en/latest/overview/changelog.html
+    https://zulip.readthedocs.io/en/stable/production/upgrade-or-modify.html
+[changelog]: https://zulip.readthedocs.io/en/stable/overview/changelog.html
```

Comment:
The changelog might want a /latest/ link, since the point we're trying to make here is in part that Zulip Cloud runs newer than the latest release.

https://zulip.readthedocs.io/en/stable/overview/changelog.html

Reply:
I think actually after looking, this link is fine, but we might want to revise "Zulip Cloud organizations are always updated to the latest version of Zulip." to either be more clear that it'll be newer than the latest stable release, and/or link to the right part of the release lifecycle documentation. (Which may need some tweaking to have an easy anchor to link to for "how Zulip Cloud works").

Maybe you can pick that up as a follow-up @drrosa?